config/defines.inc.php

```diff
@@ -67,9 +67,7 @@
 
 define('_PS_ALL_THEMES_DIR_', _PS_ROOT_DIR_.'/themes/');
 /* BO THEMES */
-if (defined('_PS_ADMIN_DIR_')) {
-    define('_PS_BO_ALL_THEMES_DIR_', _PS_ADMIN_DIR_.'/themes/');
-}
+define('_PS_BO_ALL_THEMES_DIR_', _PS_ADMIN_DIR_.'/themes/');
```

Comment:
Now  ̀_PS_ADMIN_DIR_` is only available in `/admin` context :trollface:

Reply:
Dammit!